### ISSUE TITLE
cellBottomLabel overlaps at Avatar

### ISSUE THREAD

**Issue by aaasifrz9** [MessageKit/MessageKit#542]
## General Information

I have resized Avatar using method

```swift
func avatarSize(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> CGSize {
    return CGSize(width: 50.0, height: 50.0)
}
```

When I type short message then `cellBottomLabel` overlaps at Avatar as shown in attached screenshot. How can I set `cellBottomLabel` to right aligned with the cell when text small. Or another way to not overlap, please help.

![img_1852](https://user-images.githubusercontent.com/24565054/36664693-76f6ea88-1b0b-11e8-9463-3505eef0d49c.PNG)

**Comment by SD10** [comment]
Could be a bug regarding the max width available for the label. 🤔 The goal is to remove the labels from the cell for MessageKit 1.0.

**Comment by neacao** [comment]
@SD10 , can u explain why need to remove `the label bottom` - the view that shows date time? 🤔

**Comment by SD10** [comment]
@neacao I want to encourage people to start putting the label in its own cell or as a header or footer view

**Comment by neacao** [comment]
i don't think it's good approach, by default, the framework should cover that task and disable by default (user can handle the delegate to override if needed) 🤔 . Might i miss something that you want to enhance inside framework 😄

**Comment by SD10** [comment]
@aaasifrz9 I think I've fixed this in MessageKit 1.0.0-beta.1. Thanks for reporting 👍 Feel free to reopen this issue if the problem persists
